src/instruments/src/EFB/Presets/Widgets/LightPresets.tsx

```diff
@@ -0,0 +1,177 @@
+                    {[1, 2, 3, 4, 5, 6].map((i) => (
+                        <SinglePreset
+                            presetID={i}
+                            getPresetName={getPresetName}
+                            storePresetName={storePresetName}
+                            namesMap={namesMap}
+                        />
+                    ))}
+                </div>
+            </ScrollableContainer>
+        </div>
+    );
+};
+
+// One line of preset with ID, name, load and save
+const SinglePreset = (props: { presetID: number, getPresetName: (arg0: number) => string, storePresetName: (arg0: number, arg1: string) => void, namesMap: any }) => {
+    // Light presets are handled in a wasm module as setting the indexed "LIGHT POTENTIOMETER"
+    // variable didn't work in Javascript.
+    // To tell the presets.wasm module to load a preset the LVAR "L:A32NX_LOAD_LIGHTING_PRESET"
+    // needs to be set with a number > 0 where the number is the corresponding preset ID to be loaded.
+    // If a preset is not defined for this number a default preset (all lights at 50%) will be loaded.
+    // To tell the presets.wasm module to save a preset the LVAR "L:A32NX_SAVE_LIGHTING_PRESET"
+    // needs to be set with a number > 0 where the number is the corresponding preset ID to be saved..
+    // After loading or saving the wasm module will reset the LVARs to 0.
+
+    const [, setLoadPresetVar] = useSimVar('L:A32NX_LOAD_LIGHTING_PRESET', 'number', 200);
+    const [, setSavePresetVar] = useSimVar('L:A32NX_SAVE_LIGHTING_PRESET', 'number', 200);
+
+    // Only allow loading and saving when aircraft is powered
+    const [isPowered] = useSimVar('L:A32NX_ELEC_AC_1_BUS_IS_POWERED', 'number', 200);
+
+    // Sets the LVAR to tell the wasm to load the preset into the aircraft
+    const loadPreset = (presetID: number) => {
+        // loading of presets only allowed when aircraft is powered (also the case in the wasm)
+        if (isPowered) {
+            setLoadPresetVar(presetID);
+            toast.success(`Loading Preset: ${presetID}: ${presetName}`, {
+                autoClose: 250,
+                hideProgressBar: true,
+                closeButton: false,
+            });
+        } else {
+            toast.warning('Aircraft needs to be powered to load presets.', {
+                autoClose: 1000,
+                hideProgressBar: true,
+                closeButton: false,
+            });
+        }
+    };
+
+    // Sets the LVAR to tell the wasm to save the current lighting setting into the preset
+    const savePreset = (presetID: number) => {
+        // Saving of presets only allowed when aircraft is powered (also the case in the wasm)
+        if (isPowered) {
+            setSavePresetVar(presetID);
+            toast.success(`Saving Preset: ${presetID}: ${presetName}`, {
+                autoClose: 250,
+                hideProgressBar: true,
+                closeButton: false,
+            });
+        } else {
```

Comment:
We might want to create an alternative UI like the one on the Aircraft Presets page that prevents users from interacting with the UI until the aircraft is powered, maybe something like an overlay?

Reply:
Good idea - will deactivate loading and saving based on AC power state.